external-docs/docs/guides/pivoted-shape.md

```diff
@@ -0,0 +1,101 @@
+## Single Measure
+
+The [standard shape](./standard-shape.md) is flexible but it also has a lot of redundancy which can often be removed by using the more concise pivoted form. Our dataset on the distribution of the number of `Arthur's Bakes' stores can be expressed in the pivoted shape as follows:
```

Comment:
The quotes around `Arthur's Bakes` don't match.

Reply:
Done